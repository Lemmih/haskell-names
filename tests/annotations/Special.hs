module Special where

foo :: ()
foo = ()
