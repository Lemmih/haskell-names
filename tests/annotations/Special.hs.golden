()       at  3:8 is none
()       at  4:7 is none
